user_agent.go

```diff
@@ -153,7 +153,7 @@ func (p *UserAgent) Parse(ua string) {
 }
 
 // Returns the mozilla version (it's how the User Agent string begins:
-// "Mozilla/5.0 ...", unless we're dealing with Opera, of course).
+// "Mozilla/5.0 ...", unless we're dealing with Opera or Dalvik, of course).
```

Comment:
For Dalvik this will return 5.0 now no? Or at least I can see that on the line 48 of browser.go